### PR TITLE
Gps tests 1

### PR DESCRIPTION
**Pull request by sunny-goel**

### CODE REVIEW

File: app/src/androidTest/java/testUtils/MyDao.java [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+import androidx.room.Dao;
+import androidx.room.Query;
+
+import com.shuttl.location_pings.data.model.entity.GPSLocation;
+
+import java.util.List;
+
+@Dao
+public interface MyDao {
```
Comment (sunny-goel) [inline app/src/androidTest/java/testUtils/MyDao.java:12]: @nikhil-roy  How can I use This to query the database ?

File: build.gradle [hunk 2]
```diff
@@ -8,7 +8,7 @@ buildscript {
         google()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.5.2'
+        classpath 'com.android.tools.build:gradle:3.6.1'
```
Comment (sunny-goel) [inline build.gradle:11]: @nikhil-roy I have updated the gradle verison. It is Ok ?

Comment (nikhil-roy) [reply build.gradle:11]: its fine

File: app/src/androidTest/java/testUtils/MyDao.java (hunk 1 above)
Comment (nikhil-roy) [reply app/src/androidTest/java/testUtils/MyDao.java:12]: by using locations() u can get all the locations

**Pull request closed without merging**